CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 * Remove now-unused `config.IsDevelopmentEnv()` helper to encourage better
   testing practices. Not a breaking change as nobody is using this func
   themselves according to our code search tool.
+* Log traceID and spanID in hex format to match jaeger UI. Upgrade Jaeger to 2.1.1 and use jaeger's
```

Comment:
also may as well back-tick traceID and `spanID` to be clear that they're code things